controllers/url.go

```diff
@@ -58,24 +66,38 @@ func CreateTinyURL(ctx *gin.Context, db *bun.DB) {
 	} else {
 		generatedShortURL := utils.GenerateMD5Hash(body.OriginalUrl)
 		var existingURL models.Tinyurl
-		if err := db.NewSelect().Model(&existingURL).Where("short_url = ?", generatedShortURL).Limit(1).Scan(ctx); err != nil {
+		if err := db.NewSelect().Model(&existingURL).
+			Where("short_url = ?", generatedShortURL).
+			Limit(1).
+			Scan(ctx); err != nil {
 			body.ShortUrl = generatedShortURL
 		}
 	}
-	count, _ := db.NewSelect().Model(&models.Tinyurl{}).Where("user_id = ?", body.UserID).Where("is_deleted=?", false).Count(ctx)
 
-	body.CreatedAt = time.Now().UTC()
+	// Count the number of existing URLs for the user
+	count, err := db.NewSelect().
+		Model(&models.Tinyurl{}).
+		Where("user_id = ?", body.UserID).
+		Where("is_deleted = ?", false).
+		Count(ctx)
 
-	intVar, err := strconv.ParseInt(config.Max_Url_Count, 0, 64)
+	if err != nil {
```

Comment:
why we can't merge #84  and #94, having mentioned that do we really need to show error mentioned in line 86 to user

Reply:
Ok I'll merge them